css/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    color: white;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+:root {
+    --white: #f3f3f3;
+    --darkblue: #1b1b32;
+    --darkblue-alpha: rgba(27, 27, 50, 0.8);
+    --green: #37af65;
+}
+
+body {
+    height: 100vh;
+    width: 100vw;
+    background-image: url(../assets/bg.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+}
+
+.bg {
+    background-image: linear-gradient(115deg,
+            rgba(58, 58, 158, 0.8),
+            rgba(136, 136, 206, 0.7));
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+#title {
```

Comment:
[OPATIONALLY] so you have done a good job :), however, I would recommend you to use CSS relative units like `em`, `rem` instead of `px`, that's will give you more responsiveness of your UI